packages/hebao_v1/contracts/modules/core/ERC1271Module.sol

```diff
@@ -47,8 +47,7 @@ abstract contract ERC1271Module is ERC1271, BaseModule
         returns (bytes4 magicValue)
     {
         address wallet = msg.sender;
-        (uint _lock,) = controllerCache.securityStore.getLock(wallet);
-        if (_lock > block.timestamp) { // wallet locked
+        if (controllerCache.securityStore.isLocked(wallet)) {
```

Comment:
Do we need to add back the old Api isValidSignature(bytes hash, bytes signature) in case of interoperability issue with old app.

Reply:
Lets not. It's too eary to priority backward compatibility.